Create ui-c8y-1021-27-0-plugins-installed-via-tag-should-also-correctly-show-up-in-the.md (#2748)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-27-0-plugins-installed-via-tag-should-also-correctly-show-up-in-the.md

---
date: ""

> title: plugins installed via tag should also correctly show up in the list (#7828)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Plugins installed via tag are now correctly displayed in the list
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60812
version: 1021.27.0
---

> plugins installed via tag should also correctly show up in the list (#7828)

Author: The Claude AI suggested to use the following body:
```suggestion
Previously, plugins that were installed using a tag were not properly displayed in the list of installed plugins, which could cause confusion for users trying to manage their plugins. With this change, any plugin installed via tag will now correctly appear in the plugins list. This improvement enhances the accuracy and reliability of the plugin management system, ensuring that users have a clear overview of all installed plugins regardless of the installation method.
```

> Previously, plugins that were installed using a tag were not properly displayed in the list of installed plugins, which could cause confusion for users trying to manage their plugins. With this change, any plugin installed via tag will now correctly appear in the plugins list. This improvement enhances the accuracy and reliability of the plugin management system, ensuring that users have a clear overview of all installed plugins regardless of the installation method.

Reviewer: ```suggestion
Previously, plugins that were installed using a tag were not properly displayed in the list of installed plugins, which could cause confusion for users trying to manage their plugins. With this change, any plugin installed via a tag will now correctly appear in the plugins list. This improvement enhances the accuracy and reliability of the plugin management system, ensuring that users have a clear overview of all installed plugins regardless of the installation method.
```
